README.md

```diff
@@ -98,23 +98,23 @@ After you're finished please remove all the comments and instructions!
 
 <!-- Features -->
 
-### Key Features <a name="key-features"></a>
+<!-- ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
 - **[key_feature_1]**
 - **[key_feature_2]**
-- **[key_feature_3]**
+- **[key_feature_3]** -->
```

Comment:
- [OPTIONAL] Please kindly add the future features of this application.

Reply:
hi @topeogunleye thank you so much for the kind review.. i will definitely add the features you suggessted.

Reply:
You're welcome @paulshaun-kdtk.